Makefile

```diff
@@ -1,2 +1,10 @@
 go-test:
 	find . -name go.mod -execdir go test ./... -race -covermode=atomic -coverprofile=coverage.out \;
+
+run-cypress-dc:
+	docker compose -f ./opg-modernising-lpa/docker-compose.yml \
```

Comment:
should these paths be prefixed with `./opg-modernising-lpa/`?

Reply:
yep, third time lucky 😄